scripts/npm/apps-install/index.js

```diff
@@ -13,25 +13,29 @@ const paths = require( '../../../config/environment' ).paths;
 // eslint-disable-next-line no-sync
 let apps = fs.readdirSync( `${ paths.unprocessed }/apps/` );
 
-// Filter out .DS_STORE directory.
+// Filter out hidden directories.
 apps = apps.filter( dir => dir.charAt( 0 ) !== '.' );
 
-// Run each application's JS through webpack and store the gulp streams.
+// Install each application's dependencies.
 apps.forEach( app => {
-  /* Check if node_modules directory exists in a particular app's folder.
-     If it doesn't log the command to add it and don't process the scripts. */
+  /* Check if package.json exists in a particular app's folder.
+     If it does, run npm install on that directory. */
   const appsPath = `${ paths.unprocessed }/apps/${ app }`;
   // eslint-disable-next-line no-sync
   if ( fs.existsSync( `${ appsPath }/package.json` ) ) {
     exec( `npm --prefix ${ appsPath } install ${ appsPath }`,
```

Comment:
Swap this out for `execSync.`. You want this to be blocking, you will need to modify the code to use `try / catch`. See https://github.com/cfpb/cfgov-refresh/pull/3840/files#diff-269192501d1bd2ea09c118162a0ab16aR29